src/c++/perf_analyzer/command_line_parser.cc

```diff
@@ -717,6 +723,13 @@ CLParser::Usage(const std::string& msg)
   exit(GENERIC_ERROR);
 }
 
+void
+CLParser::PrintVersion()
+{
+  std::cerr << "Perf Analyzer Version: " << VERSION << std::endl;
```

Comment:
nit: `Perf Analyzer version`, and no `:`

Reply:
I am modeling this after the triton server output from the container start up. I removed the colon, kept the capitalization.

![Screenshot 2023-06-21 at 09 36 45](https://github.com/triton-inference-server/client/assets/6505145/7ef2f7f1-792c-4fc2-9842-caa76cfee5c7)